### PR TITLE
7988 E2E-tester for 25%-regel og minstebeløp, EU/EØS pensjonist

### PR DESCRIPTION
**Pull request by ragnarwestad**
## Summary
E2E-tester for MELOSYS-7988: Visning av trygdeavgift ved 25%-regel og minstebeløp for **EU/EØS Pensjonist**.

- **AC1**: Infomelding når inntekt er under minstebeløpet
- **AC2**: Tabell med `*` i sats-kolonnen ved 25%-regel
- **AC4**: `***` i inntektskilde-kolonnen ved sammenslåtte inntektskilder
- Regresjonstest: Ordinær beregning viser tabell uten spesialmeldinger

> **AC3** (Helsedel/Pensjonsdel i Dekning-kolonnen) gjelder FTRL-flyten. EU/EØS-pensjonist-tabellen har ingen Dekning-kolonne (`{!erEøsPensjonist && <HeaderCell>Dekning</HeaderCell>}`), så AC3 hører ikke hjemme her. Den er allerede dekket av #242 (`tests/ftrl/ftrl-trygdeavgift-25-prosent-regel.spec.ts` — `verifiserDekningKolonne` Helsedel/Pensjonsdel). Derfor legges ingen egen FTRL-helsedel/pensjonsdel-test til i denne PR-en.

Tester for https://jira.adeo.no/browse/MELOSYS-7988

Ikke merge inn før https://github.com/navikt/melosys-web/pull/3041 er merget.

## Nye filer
- `pages/behandling/eu-eos-pensjonist-inngang.page.ts` + `eu-eos-pensjonist-inngang.assertions.ts` — POM for Inngang-steget
- `pages/trygdeavgift/eu-eos-pensjonist-trygdeavgift.page.ts` + `eu-eos-pensjonist-trygdeavgift.assertions.ts` — POM for Trygdeavgift-steget
- `tests/eu-eos/eu-eos-pensjonist-trygdeavgift.spec.ts` — AC1, AC2, AC4 + regresjonstest

## Test plan
- [x] Alle 4 tester kjører grønt lokalt
- [ ] CI-kjøring bekrefter at testene passerer

### CODE REVIEW

File: pages/trygdeavgift/eu-eos-pensjonist-trygdeavgift.page.ts [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+  async fyllInnBruttoinntektForIndeksMedApiVent(index: number, beløp: string): Promise<void> {
+    await this.ventPåBeregning(async () => {
+      const felt = this.bruttoinntektForIndeks(index);
+      await felt.click();
+      await felt.fill(beløp);
+      await felt.press('Tab');
+    });
+  }
+
+  async verifiserInfomeldingMinstebeløpSynlig(): Promise<void> {
```
Comment (RuneLind) [inline pages/trygdeavgift/eu-eos-pensjonist-trygdeavgift.page.ts]: **Konvensjon: `verifiser*`-metoder hører hjemme i `*.assertions.ts`**

SKILL.md (`pom-from-recording`) og eksisterende POMs (se `trygdeavgift.assertions.ts`) skiller actions (Page) fra verifiseringer (Assertions). De fire `verifiser*`-metodene under bør flyttes til en ny `eu-eos-pensjonist-trygdeavgift.assertions.ts` og eksponeres via `readonly assertions` på Page.

Bruken i tester blir da `trygdeavgift.assertions.verifiserInfomeldingMinstebeløpSynlig()` — konsistent med resten av repoet.

File: pages/behandling/eu-eos-pensjonist-inngang.page.ts [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+ * - Fylle inn helseutgiftDekkesPeriode (fom/tom)
+ * - Velge bostedsland
+ * - Bekrefte og gå videre til Trygdeavgift-steget
+ *
+ * @example
+ * const inngang = new EuEosPensjonistInngangPage(page);
+ * await inngang.ventPåSideLastet();
+ * await inngang.fyllInnPeriode('01.01.2024', '31.12.2024');
+ * await inngang.velgBostedsland('SE');
+ * await inngang.klikkBekreftOgFortsett();
```
Comment (RuneLind) [inline pages/behandling/eu-eos-pensjonist-inngang.page.ts:19]: **Mangler tilhørende `*.assertions.ts`-fil**

For konsistens med konvensjonen (se `eu-eos-behandling.assertions.ts`, `medlemskap.assertions.ts` osv.), bør denne POM-en ha en assertions-fil — selv om den foreløpig kun trenger `verifiserSideLastet()`. `ventPåSideLastet()` her er teknisk sett en assertion som heter "vent på".

File: tests/eu-eos/eu-eos-pensjonist-trygdeavgift.spec.ts [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+    await inngang.klikkBekreftOgFortsett();
+
+    const trygdeavgift = new EuEosPensjonistTrygdeavgiftPage(page);
+    await trygdeavgift.ventPåSideLastet();
+    await trygdeavgift.velgIkkeSkattepliktig();
+    await trygdeavgift.velgInntektskilde('PENSJON');
+    await trygdeavgift.fyllInnBruttoinntektMedApiVent('9000');
+
+    await trygdeavgift.verifiserTrygdeavgiftsTabellSynlig();
+    await expect(page.getByRole('cell', { name: '*' })).toBeVisible();
```
Comment (RuneLind) [inline tests/eu-eos/eu-eos-pensjonist-trygdeavgift.spec.ts]: **Inline `expect()` bør flyttes til POM-assertions**

```ts
await expect(page.getByRole('cell', { name: '*' })).toBeVisible();
await expect(page.getByText('* Beregnet etter 25 %-regelen')).toBeVisible();
```

Foreslår: `await trygdeavgift.assertions.verifiser25ProsentRegelMarkering()`. Da kan UI-tekstendringer (f.eks. ny fotnote-formulering) oppdateres på ett sted istedenfor å spore opp inline-asserts i flere tester.

File: tests/eu-eos/eu-eos-pensjonist-trygdeavgift.spec.ts [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+    // Samlet 2×5000×12=120000 > minstebeløpet, men avgift > 25% av overskudd → sammenslåing
+    await trygdeavgift.klikkLeggTilInntekt();
+    await trygdeavgift.velgInntektskildeForIndeks(1, 'Uføretrygd');
+    await trygdeavgift.fyllInnBruttoinntektForIndeksMedApiVent(1, '5000');
+
+    await trygdeavgift.verifiserTrygdeavgiftsTabellSynlig();
+    await expect(page.getByRole('cell', { name: '***' })).toBeVisible();
+    await expect(page.getByText('*** Mer enn en inntektskilde')).toBeVisible();
+  });
+
```
Comment (RuneLind) [inline tests/eu-eos/eu-eos-pensjonist-trygdeavgift.spec.ts:154]: **Samme som over — flytt til assertions**

```ts
await expect(page.getByRole('cell', { name: '***' })).toBeVisible();
await expect(page.getByText('*** Mer enn en inntektskilde')).toBeVisible();
```

Foreslår: `verifiserSammenslåtteInntektskilderMarkering()` i `eu-eos-pensjonist-trygdeavgift.assertions.ts`.

File: pages/trygdeavgift/eu-eos-pensjonist-trygdeavgift.page.ts [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+    await expect(
+      this.page.getByText('Trygdeavgift skal ikke betales da inntekten er under minstebeløpet.'),
+    ).not.toBeVisible();
+  }
+
+  async verifiserTrygdeavgiftsTabellSynlig(): Promise<void> {
+    await expect(this.page.getByRole('columnheader', { name: 'Trygdeperiode' })).toBeVisible();
+  }
+
+  async verifiserTrygdeavgiftsTabellIkkeSynlig(): Promise<void> {
```
Comment (RuneLind) [inline pages/trygdeavgift/eu-eos-pensjonist-trygdeavgift.page.ts]: **Mulig race condition i `verifiserTrygdeavgiftsTabellIkkeSynlig`**

`.not.toBeVisible()` returnerer umiddelbart true hvis tabellen ikke er der akkurat nå — men sier ingenting om hvorvidt React kan rendre den 50ms senere.

I testen kalles denne etter `verifiserInfomeldingMinstebeløpSynlig`, som venter på at meldingen kommer — så i praksis er det greit. Men metoden alene er ikke robust. Vurder å la den vente på at infomeldingen er synlig først, eller dokumenter forutsetningen i metoden.

File: tests/eu-eos/eu-eos-pensjonist-trygdeavgift.spec.ts [hunk 6]
```diff
@@ -0,0 +1,154 @@
...
+    await inngang.klikkBekreftOgFortsett();
+
+    const trygdeavgift = new EuEosPensjonistTrygdeavgiftPage(page);
+    await trygdeavgift.ventPåSideLastet();
+    await trygdeavgift.velgIkkeSkattepliktig();
+
+    // Første inntektskilde: Pensjon
+    await trygdeavgift.velgInntektskilde('PENSJON');
+    await trygdeavgift.fyllInnBruttoinntektMedApiVent('5000');
+
```
Comment (RuneLind) [inline tests/eu-eos/eu-eos-pensjonist-trygdeavgift.spec.ts:146]: **G-avhengige testdata — risiko for fremtidig flakiness**

Kommentaren forklarer matematikken godt, men minstebeløpet og 25%-regelens grenser er G-justerte og endres årlig. Hvis G-justeringen flytter terskelen forbi `2×5000×12 = 120000`, vil testen plutselig feile uten at noe i koden har endret seg.

Forslag: Definer disse som konstanter med kommentar om når de må revurderes (f.eks. `MIN_INNTEKT_FOR_25_REGEL = 5000`), eller bruk verdier med god margin på begge sider.

File: tests/ftrl/ftrl-trygdeavgift-helsedel-pensjonsdel.spec.ts [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+import { OpprettNySakPage } from '../../pages/opprett-ny-sak/opprett-ny-sak.page';
+import { MedlemskapPage } from '../../pages/behandling/medlemskap.page';
+import { ArbeidsforholdPage } from '../../pages/behandling/arbeidsforhold.page';
+import { LovvalgPage } from '../../pages/behandling/lovvalg.page';
+import { ResultatPeriodePage } from '../../pages/behandling/resultat-periode.page';
+import { TrygdeavgiftPage } from '../../pages/trygdeavgift/trygdeavgift.page';
+import { AARSAK, SAKSTEMA, SAKSTYPER, USER_ID_VALID } from '../../pages/shared/constants';
+import { waitForProcessInstances } from '../../helpers/api-helper';
+
+const inneværendeÅr = new Date().getFullYear();
```
Comment (RuneLind) [inline tests/ftrl/ftrl-trygdeavgift-helsedel-pensjonsdel.spec.ts]: **`new Date().getFullYear()` — kalenderårskifte-risiko**

Dynamisk inneværende år er greit lokalt, men:
- Fra 1. januar et nytt år vil testene plutselig kjøre mot et år hvor mock-data eller G-verdier kanskje ikke er satt opp
- Saker som krysser årsskiftet (rundt 31. desember) kan oppføre seg ulikt enn forventet

Lav prioritet, men vurder å låse til et år (f.eks. `2026`) med god mock-data-dekning.

**Review by RuneLind — COMMENTED** [review]
Hovedfunn: `verifiser*`-metoder bør flyttes til `*.assertions.ts` for konsistens med POM-konvensjonen, og inline `expect()` i testene bør samles i assertions-metoder. Resten er watchpoints (G-avhengige testdata, kalenderårskifte).

**Comment by RuneLind** [comment]
@copilot merge in main

File: pages/behandling/eu-eos-pensjonist-inngang.assertions.ts [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+/**
+ * Assertion-metoder for EuEosPensjonistInngangPage
+ *
+ * Ansvar:
+ * - Verifisere at siden er lastet korrekt
+ */
+export class EuEosPensjonistInngangAssertions {
+  constructor(readonly page: Page) {}
+
+  async verifiserSideLastet(): Promise<void> {
```
Comment (RuneLind) [inline pages/behandling/eu-eos-pensjonist-inngang.assertions.ts:12]: `verifiserSideLastet` brukes aldri — testene kaller `inngang.ventPåSideLastet()` på POM-en i stedet. Enten kall denne fra testene (skill-en anbefaler at assertion-er går via `.assertions.verifiser...()`), eller fjern den. Dead code i en helt ny fil bør unngås.

File: pages/trygdeavgift/eu-eos-pensjonist-trygdeavgift.page.ts [hunk 9]
```diff
@@ -0,0 +1,133 @@
...
+    const navn = erSkattepliktig ? 'Ja' : 'Nei';
+    // Radio-klikket trigger ikke alltid en PUT mot /eos-pensjonist/beregning
+    // (form-useEffect avhenger av antall perioder, ikke skatteplikttype-verdien).
+    // Tolerant vent: fang evt. debounced PUT, ellers fall tilbake til kort pause.
+    // Den definitive PUT'en garanteres av fyllInnBruttoinntektMedApiVent senere.
+    const beregningResponsePromise = this.page
+      .waitForResponse(
+        (resp) =>
+          resp.url().includes('eos-pensjonist/beregning') &&
+          resp.request().method() === 'PUT' &&
```
Comment (RuneLind) [inline pages/trygdeavgift/eu-eos-pensjonist-trygdeavgift.page.ts:69]: Mindre robust enn eksisterende `TrygdeavgiftPage.velgSkattepliktig` (`pages/trygdeavgift/trygdeavgift.page.ts:91-148`), som bruker `radioInput.click({ force: true })`, `expect(radio).toBeChecked()`-polling og label-klikk-fallback. På CI har vi sett radio-klikk som ikke registreres (jf. step-transition-bug). Vurder å gjenbruke samme pattern her for å unngå flakiness.

`waitForTimeout(1500)` på linje under er en arbitrær venting som skjuler symptomet i stedet for å vente på faktisk state.

File: pages/trygdeavgift/eu-eos-pensjonist-trygdeavgift.page.ts [hunk 10]
```diff
@@ -0,0 +1,133 @@
...
+
+    const response = await beregningResponsePromise;
+    if (!response) {
+      await this.page.waitForTimeout(1500);
+    }
+  }
+
+  async velgInntektskilde(inntektskildetype: string): Promise<void> {
+    await this.inntektskildeDropdown.waitFor({ state: 'visible', timeout: 15000 });
+    await expect(this.inntektskildeDropdown).toBeEnabled();
```
Comment (RuneLind) [inline pages/trygdeavgift/eu-eos-pensjonist-trygdeavgift.page.ts:91]: Inkonsistent API: denne tar dropdown-**value** (`'PENSJON'`), mens `velgInntektskildeForIndeks` på linje 119 tar **visible label** (`'Uføretrygd'`). Det blir forvirrende for senere lesere.

Eksisterende `TrygdeavgiftPage.velgInntektskilde` (`trygdeavgift.page.ts:166-237`) har en `labelMapping` som håndterer begge formater. Anbefaling: bruk én konvensjon (helst value-kodene) og legg `PENSJON`/`UFØRETRYGD` til en ny `INNTEKTSKILDE`-konstant i `pages/shared/constants.ts`.

File: pages/trygdeavgift/eu-eos-pensjonist-trygdeavgift.assertions.ts [hunk 11]
```diff
@@ -0,0 +1,62 @@
...
+
+  /**
+   * Verifiserer at beregningstabellen ikke er synlig.
+   *
+   * Forutsetning: Kall etter `verifiserInfomeldingMinstebeløpSynlig()` for å sikre
+   * at React har ferdig-rendret siden (infomeldingen garanterer at beregningen er fullført).
+   * Uten denne forutsetningen kan assertion-en bestå for tidlig, siden assertionstimeout
+   * kan utløpe før tabellen eventuelt dukker opp i en sen React-render.
+   */
+  async verifiserTrygdeavgiftsTabellIkkeSynlig(): Promise<void> {
```
Comment (RuneLind) [inline pages/trygdeavgift/eu-eos-pensjonist-trygdeavgift.assertions.ts]: `not.toBeVisible()` bruker default assertion-timeout — kan passere for tidlig før tabellen *eventuelt* dukker opp i en sen React-render. Doc-kommentaren erkjenner dette, men API-en kan ikke håndheve forutsetningen.

Forslag:
- Hardere: `await expect(this.page.getByRole('columnheader', { name: 'Trygdeperiode' })).toHaveCount(0);` (failer hvis den dukker opp innen default-timeout).
- Eller slå sammen `verifiserInfomeldingMinstebeløpSynlig` + denne til én `verifiserMinstebeløpModus()` som håndhever rekkefølgen.

File: tests/eu-eos/eu-eos-pensjonist-trygdeavgift.spec.ts [hunk 12]
```diff
@@ -0,0 +1,164 @@
...
+  test('skal vise infomelding når inntekten er under minstebeløpet', async ({ page, request }) => {
+    test.setTimeout(120000);
+
+    await opprettEøsPensjonistTrygdeavgiftSak(page, request);
+
+    const inngang = new EuEosPensjonistInngangPage(page);
+    await inngang.ventPåSideLastet();
+    await inngang.fyllInnPeriode(helÅrFra, helÅrTil);
+    await inngang.velgBostedsland('SE');
+    await inngang.klikkBekreftOgFortsett();
```
Comment (RuneLind) [inline tests/eu-eos/eu-eos-pensjonist-trygdeavgift.spec.ts:81]: Hardkodet ISO-kode `'SE'` gjentas i alle fire testene. `pages/shared/constants.ts:53-58` har allerede `ARBEIDSLAND.SWEDEN = 'SE'` — bruk konstanten for konsistens:

```ts
import { ARBEIDSLAND } from '../../pages/shared/constants';
await inngang.velgBostedsland(ARBEIDSLAND.SWEDEN);
```

Gjelder også linje 101, 124 og 153.

File: tests/ftrl/ftrl-trygdeavgift-helsedel-pensjonsdel.spec.ts [hunk 13]
```diff
@@ -0,0 +1,93 @@
...
+  const resultatPeriode = new ResultatPeriodePage(page);
+
+  // Opprett FTRL-sak
+  await hovedside.goto();
+  await hovedside.klikkOpprettNySak();
+  await opprettSak.fyllInnBrukerID(USER_ID_VALID);
+  await opprettSak.velgSakstype(SAKSTYPER.FTRL);
+  await opprettSak.velgSakstema(SAKSTEMA.MEDLEMSKAP_LOVVALG);
+  await opprettSak.velgBehandlingstema('YRKESAKTIV');
+  await opprettSak.velgAarsak(AARSAK.SØKNAD);
```
Comment (RuneLind) [inline tests/ftrl/ftrl-trygdeavgift-helsedel-pensjonsdel.spec.ts]: Inkonsistent med EU/EØS-testen (som bruker `BEHANDLINGSTEMA.PENSJONIST`). Bytt til `BEHANDLINGSTEMA.YRKESAKTIV` — du importerer allerede `BEHANDLINGSTEMA`-relaterte konstanter, og det er én enum-utvidelse i denne PR-en.

File: tests/ftrl/ftrl-trygdeavgift-helsedel-pensjonsdel.spec.ts [hunk 14]
```diff
@@ -0,0 +1,93 @@
...
+  await opprettSak.klikkOpprettNyBehandling();
+
+  await waitForProcessInstances(page.request, 30);
+  await hovedside.goto();
+  await page.getByRole('link', { name: 'TRIVIELL KARAFFEL -' }).click();
+
+  // Medlemskap med helse- og pensjonsdel
+  await medlemskap.velgPeriode(helÅrFra, helÅrTil);
+  await medlemskap.velgLand('Afghanistan');
+  await medlemskap.velgTrygdedekning('FTRL_2_9_FØRSTE_LEDD_C_HELSE_PENSJON');
```
Comment (RuneLind) [inline tests/ftrl/ftrl-trygdeavgift-helsedel-pensjonsdel.spec.ts]: Mangler `await page.waitForLoadState('networkidle');` etter `TRIVIELL KARAFFEL`-klikket. EU/EØS-testen har det (`eu-eos-pensjonist-trygdeavgift.spec.ts:47`) — uten samme settling-vent kan denne bli mer flaky på CI når behandlingen lastes inn.

**Review by RuneLind — COMMENTED** [review]
Solid POM-tester for MELOSYS-7988. Se inline-kommentarer — hovedpoeng: gjenbruk eksisterende robust radio-klikk-pattern, unifiser `velgInntektskilde`-API-et, og rydd opp i hardkodede strenger som har konstanter. Resten er polish.

File: tests/eu-eos/eu-eos-pensjonist-trygdeavgift.spec.ts [hunk 15]
```diff
@@ -0,0 +1,169 @@
...
+ * Tester beregning av trygdeavgift for EU/EØS Pensjonist-saker
+ * mot ekte backend (ingen mocking).
+ *
+ * Dekker:
+ * - Infomelding "Trygdeavgift skal ikke betales..." når alle perioder er under minstebeløpet
+ * - 25%-regel → tabell med * i sats-kolonnen og fotnote
+ * - Sammenslåtte inntektskilder → *** i inntektskilde-kolonnen og fotnote
+ * - Regresjon: Ordinær beregning → tabell synlig, ingen infomelding
+ */
+test.describe('EU/EØS Pensjonist - Trygdeavgift beregningsresultat', () => {
```
Comment (RuneLind) [inline tests/eu-eos/eu-eos-pensjonist-trygdeavgift.spec.ts:66]: **Feil sak-referanse?** PR-body sier «Tester for MELOSYS-7988», men [MELOSYS-7988](https://jira.adeo.no/browse/MELOSYS-7988) gjelder *årsavregning*. Disse testene kjører **trygdeavgiftssteget** for `Sakstema=TRYGDEAVGIFT, Sakstype=EU_EOS, Behandlingstema=PENSJONIST` — som er eksplisitt listet som én av fire stegvelgere i [MELOSYS-7989](https://jira.adeo.no/browse/MELOSYS-7989) «Tilpasse visning i trygdeavgiftssteget» (du har selv kommentert på den saken). Bør PR-en knyttes til 7989?

File: tests/eu-eos/eu-eos-pensjonist-trygdeavgift.spec.ts [hunk 16]
```diff
@@ -0,0 +1,169 @@
...
+  test('skal vise infomelding når inntekten er under minstebeløpet', async ({ page, request }) => {
```
Comment (RuneLind) [inline tests/eu-eos/eu-eos-pensjonist-trygdeavgift.spec.ts:67]: **Dekningsgap mot AC.** Denne testen dekker *hele året under minstebeløpet* → infomelding + skjult tabell. Det er et gyldig edge-case, men AC-en beskriver i tillegg en egen visning: `**` i sats-kolonnen med fotnote *«**Inntekten er under minstebeløpet»* for perioder under minstebeløpet (der tabellen fortsatt vises). Den `**`-stien er ikke dekket av noen test her. Verdt å avklare om den skal med, eller om den hører hjemme i en egen test/sak.

File: tests/eu-eos/eu-eos-pensjonist-trygdeavgift.spec.ts [hunk 17]
```diff
@@ -0,0 +1,169 @@
...
+// Låst til 2026 for å unngå G-avhengig flakiness ved kalenderårsskifte.
+const TESTÅR = 2026;
+const helÅrFra = `01.01.${TESTÅR}`;
+const helÅrTil = `31.12.${TESTÅR}`;
+
+async function opprettEøsPensjonistTrygdeavgiftSak(page: Page, request: APIRequestContext) {
+  // Cleanup-fixturen resetter alle Unleash-toggles før hver test;
+  // 25%-regelen må derfor enables eksplisitt her.
+  const unleash = new UnleashHelper(request);
+  await unleash.enableFeature('melosys.trygdeavgift.25-prosentregel');
```
Comment (RuneLind) [inline tests/eu-eos/eu-eos-pensjonist-trygdeavgift.spec.ts:30]: Felleshjelperen skrur alltid **på** `melosys.trygdeavgift.25-prosentregel`, så regresjonstesten «skal vise tabell ved ordinær beregning» verifiserer «ordinær inntekt → tabell uten `*`» *med feature-en på* — ikke toggle-av-stien. Det matcher AC-en (visningslogikk), så greit å la stå, men da er det ikke en ekte toggle-av-regresjon.

Merk også: toggelen må være opprettet av melosys-api-imaget CI bruker (auto-opprettes ved oppstart), ellers feiler `enableFeature` her.

File: tests/eu-eos/eu-eos-pensjonist-trygdeavgift.spec.ts [hunk 18]
```diff
@@ -0,0 +1,169 @@
...
+
+  // 25%-regel → tabell med * i sats-kolonnen og fotnote
+  test('skal vise tabell med asterisk (*) for 25%-regel', async ({ page, request }) => {
+    test.setTimeout(120000);
+
+    // Litt over minstebeløpet, men lav nok til at 25%-regelen begrenser avgiften.
+    // Ratio 1.085 er valgt for å speile det forholdet som fungerte med tidligere
+    // hardkodet verdi (9000 kr/md ved minstebeløp ~99k/år).
+    const månedligMinstebeløp = Math.floor((await hentMinstebeløp(request, TESTÅR)) / 12);
+    const månedsinntektFor25ProsentRegel = String(Math.floor(månedligMinstebeløp * 1.085));
```
Comment (RuneLind) [inline tests/eu-eos/eu-eos-pensjonist-trygdeavgift.spec.ts:100]: Magisk ratio `1.085` (og `0.6` for sammenslåtte kilder lenger ned). Godt kommentert, men den kobler testen til backendens nåværende sats/terskel-matematikk — endres terskel-logikken kan ratioen stille slutte å trigge 25%-regelen, og testen asserterer mot feil scenario uten å feile åpenbart. Regelen/tersklene er dokumentert i Confluence ([25-prosentregelen og minstebeløpet](https://confluence.adeo.no/spaces/TEESSI/pages/704156896) + [worked examples](https://confluence.adeo.no/spaces/TEESSI/pages/535938349)) — verdt å referere den i en kommentar, så er forutsetningen sporbar.

File: pages/trygdeavgift/eu-eos-pensjonist-trygdeavgift.assertions.ts [hunk 19]
```diff
@@ -0,0 +1,62 @@
...
+    await expect(this.page.getByRole('columnheader', { name: 'Trygdeperiode' })).not.toBeVisible();
+  }
+
+  /**
+   * Verifiserer 25%-regel-markering:
+   * - `*` i sats-kolonnen
+   * - fotnote "* Beregnet etter 25 %-regelen"
+   */
+  async verifiser25ProsentRegelMarkering(): Promise<void> {
+    await expect(this.page.getByRole('cell', { name: '*' })).toBeVisible();
```
Comment (RuneLind) [inline pages/trygdeavgift/eu-eos-pensjonist-trygdeavgift.assertions.ts:53]: `getByRole('cell', { name: '*' })` bruker som standard **substring**-matching, ikke eksakt. To konsekvenser:

1. Hvis tabellen rendrer **flere rader** med `*` i sats-kolonnen, treffer locatoren flere elementer → `toBeVisible()` kaster strict-mode-violation.
2. `name: '*'` matcher teknisk sett også en celle som inneholder `***` (substring).

Forslag: `getByRole('cell', { name: '*', exact: true }).first()`. Samme gjelder `'***'` på linje 59.

**Review by RuneLind — COMMENTED** [review]
Solid og idiomatisk testkode (POM, fixtures, live-henting av minstebeløp). Tre ting før merge, se inline: (1) PR-en peker til MELOSYS-7988 (årsavregning), men testene dekker trygdeavgiftssteget = MELOSYS-7989 — bør sak-referansen rettes? (2) AC3 helsedel/pensjonsdel + `tests/ftrl/...helsedel-pensjonsdel.spec.ts` er nevnt i beskrivelsen, men fila er ikke med — legg til eller trim beskrivelsen. (3) `**`-markeringen «Inntekten er under minstebeløpet» (per-periode) er udekket.
